src/libraries/System.Data.Odbc/ref/System.Data.Odbc.ODBC32.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Data.Odbc
+{
+    // We need this class in the implementation browser assembly for building the system.data facade.
```

Comment:
If we need this class in the implementation assembly, why is it then included in the ref assembly? Maybe you want to clarify in the comment?

Reply:
We were earlier exposing it specifically in browser implementation, later on we decided it expose in ref as well. i will remove the comment